docker-hub/scim.md

```diff
@@ -0,0 +1,65 @@
+---
+description: System for Cross-domain Identity Maagement
+keywords: SCIM, SSO
+title: System for Cross-domain Identity Management
+---
+
+> **Important**
+>
+> SCIM for Okta early access is now available. SCIM support for Azure AD is
+> coming soon.
+{: .important}
+
+This section is for administrators who want to enable System for Cross-domain Identity Management (SCIM) 2.0 for their business. SCIM is a provisioning system that lets you manage users within your identity provider (IdP). You can enable SCIM on organizations that are part of the Docker Business subscription. To upgrade your existing account to a Docker Business subscription, see Upgrade your subscription.
```

Comment:
Looks like link to upgrade your subscription is missing. We can do a follow-up PR to fix this